server/credentialstore.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Craig MacFarlane
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import redis
+import constants
+
+class CredentialStore():
+
+    def __init__(self):
+        self.red = redis.Redis(host=constants.REDIS_IP_ADDR, port=6379)
```

Comment:
Is the port always 6379? Even on production servers?

Reply:
6379 is the default port for redis servers.  And I've never seen a redis server running on anything other than 6379, including the redis servers used for ftc-scoring and all the local development servers I've run.  I think the only reason to specifically define a port other than 6379 is if you want to run two redis servers on the same machine.  I think this is fine for now.  I can add it to constants if you really want, but that's one more thing to break if not set properly.